Configurations/10-main.conf

```diff
@@ -1375,6 +1375,12 @@ sub vms_info {
         perlasm_scheme   => "auto",
         multilib         => "-x64",
     },
+    "VC-WINCORE64A" => {
+        inherit_from     => [ "VC-WIN64A" ],
+        cflags           => add("-DOPENSSL_SYS_WIN_CORE  -DOPENSSL_NO_CAPIENG -DOPENSSL_NO_ASYNC"),
+        lflags           => add("/NODEFAULTLIB:kernel32.lib /NODEFAULTLIB:ws2_32.lib /NODEFAULTLIB:gdi32.lib /NODEFAULTLIB:advapi32.lib /NODEFAULTLIB:crypt32.lib /NODEFAULTLIB:user32.lib /NXCOMPAT /DYNAMICBASE /machine:X64"),
```

Comment:
I don't quite follow. I reckon that /nodefaultlib:kernel32 would be sufficient. And that would be needed rather to shortcut references from VCRT. As mentioned elsewhere, even if you link with onecore[_downlevel], VCRT apparently leaves direct [and undesired in the context] reference to kernel32, but you can resolve it with /nodefaultlib:kernel32.

Reply:
Does one actually need /machine? Can link on current %PATH% even generate binary for another arch? Why /nxcompat? It's documented to be on by default. As so is /dynamicbase...